<ui version="4.0" >
 <class>PlayerWidget</class>
 <widget class="QWidget" name="PlayerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>125</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="horizontalSpacing" >
    <number>-1</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>_("Rack:")</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="lineEditRack" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QPushButton" name="pushButtonShuffle" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>_("Shuffle")</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>_("Play a word:")</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="lineEditPlay" >
     <property name="toolTip" >
      <string>_("To play a word, enter the word (case sensitive) and its coordinates. E.g.: WORD H4")</string>
     </property>
     <property name="statusTip" >
      <string>_("To play a word, enter the word (case sensitive) and its coordinates. E.g.: WORD H4")</string>
     </property>
     <property name="text" >
      <string>dummy</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="pushButtonPlay" >
     <property name="text" >
      <string>_("Play")</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="labelChange" >
     <property name="text" >
      <string>_("Change letters:")</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="lineEditChange" >
     <property name="toolTip" >
      <string>_("Enter the letters you want to change")</string>
     </property>
     <property name="text" >
      <string>dummy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pushButtonChange" >
     <property name="text" >
      <string>_("Change")</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="pushButtonPass" >
     <property name="text" >
      <string>_("Pass")</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
